plugin/ocgrpc/trace_test.go

```diff
@@ -231,3 +235,71 @@ func checkSpanData(t *testing.T, s1, s2 *trace.SpanData, methodName string, succ
 		t.Errorf("Got HasRemoteParent=%t, want true", s2.HasRemoteParent)
 	}
 }
+
+type testProp struct {
+	t *testing.T
+}
+
+func (p *testProp) InjectSpanContext(sc trace.SpanContext, appendHeader func(string, string)) {
+	traceHdr, err := json.Marshal(sc)
+	if err != nil {
+		p.t.Fatal(err)
+	}
+	appendHeader("x-json-trace", hex.EncodeToString(traceHdr))
```

Comment:
to be slightly more realistic, might want to make an example either b3 or similarly multi-header. just to exercise the gears (even if only in the test tree)